tldw_Server_API/app/api/v1/endpoints/media/listing.py

```
@@ -0,0 +1,776 @@
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Error performing metadata search",
+        ) from exc
+
+
+async def _validate_identifier_query(
+    doi: Optional[str] = Query(None),
+    pmid: Optional[str] = Query(None),
+    pmcid: Optional[str] = Query(None),
+    arxiv_id: Optional[str] = Query(None),
+    s2_paper_id: Optional[str] = Query(None),
```

Ruff (0.14.4)

Unused function argument: `s2_paper_id`

(ARG001)